src/ConnectCore.js

```diff
@@ -108,7 +108,7 @@ class ConnectCore {
    */
   getProvider () {
     const subProvider = new UportSubprovider({
-      requestAddress: () => this.requestCredentials({accountType: 'keypair'})
+      requestAddress: () => this.requestCredentials({accountType: this.accountType || 'keypair'})
```

Comment:
This is good except for the case where `this.accountType` is `none`. So try that and it should be good.

Reply:
@rmw2 also circleci is failing